tests/console/commands/test_add.py

```diff
@@ -24,10 +24,18 @@ def old_tester(tester):
     return tester
 
 
-def test_add_no_constraint(app, repo, tester):
+@pytest.fixture()
+def repo_with_packages(repo):
```

Comment:
Since this is more an action than an object, I'd more inclined to name it something along the lines of `repo_add_default_packages` or `add_default_packages_to_repo`  or similar. Just a personal preference.

Reply:
Agreed, this makes sense. I will update.